src/main/java/frc/utils/controllers/keyboard/KeyboardController.java

```diff
@@ -129,7 +129,7 @@ public KeyboardController() {
     }
 
     private Trigger getNewKey(String name) {
-        return new Trigger(new LoggedTableBoolean(KEYBOARD_TABLE, KEYS_TAB + name, false)::get);
+        return new Trigger(new LoggedTableBoolean(KEYBOARD_TABLE, KEYS_TAB + name)::get);
```

Comment:
why this is solving it?

Reply:
this is not solving it, i just changed it because there is a default constructor the puts the false in.

Reply:
I prefer u keep the false here if the current constructor call the same one with false,
I think it’s more readable